src/test/java/com/snack/news/repository/NewsRepositoryTest.java

```diff
@@ -203,4 +207,31 @@
 		assertThat(middleNewsPage.get().count(), equalTo((long) pageSize));
 		assertThat(lastNewsPage.get().count(), equalTo((totalElements % pageSize)));
 	}
+
+	@Test
+	@Transactional
+	public void 뉴스리스트를_생성일_역순으로_정렬할_수_있다() {
+		final int pageSize = 5;
+		List<News> expectedListFistNewsPage = newsRepository.findAll()
+				.stream().sorted(Comparator.comparing(News::getId).reversed()).limit(pageSize).collect(toList());
+
+		long lastNewsId1 = expectedListFistNewsPage.get(0).getId();
+		long lastNewsId2 = expectedListFistNewsPage.get(1).getId();
+		long lastNewsId3 = expectedListFistNewsPage.get(2).getId();
+		long lastNewsId4 = expectedListFistNewsPage.get(3).getId();
+		long lastNewsId5 = expectedListFistNewsPage.get(4).getId();
+		
+		Pageable pageableDesc = PageRequest.of(0, pageSize, new Sort(Sort.Direction.DESC, "id"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused local variables such as 'pageableDesc'.](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4508329)